esp/applications/CMakeLists.txt

```diff
@@ -13,7 +13,10 @@
 #    See the License for the specific language governing permissions and
 #    limitations under the License.
 ################################################################################
+HPCC_ADD_SUBDIRECTORY (common)
 HPCC_ADD_SUBDIRECTORY (eclwatch)
 HPCC_ADD_SUBDIRECTORY (eclservices)
 HPCC_ADD_SUBDIRECTORY (eclqueries)
+HPCC_ADD_SUBDIRECTORY (esdl)
+HPCC_ADD_SUBDIRECTORY (esdl-sandbox)
```

Comment:
This is a step in the direction I am picturing kubernetes ESDL going.  I'm promoting the use of derived containers as the source of the bindings instead of DALI.  This will be much more track-able, manageable, version-able, etc, and follow more of a config as code model. 
Using DALI for the dynamic part made sense in bare metal, but I hope the containerized way ends up even better.
esdl-sandbox will still be great for development and some production.  The name is meant as sort of a deterrent but if it remains popular I can always clone it into a more production friendly name in the future.  The sandbox is deployed by default to a simple system because it's great for developers.
All of this is currently sort of a technical preview so we can work out the kinks of both models and see what we like best.

Reply:
Got it, makes sense.